### PR TITLE
setup repo and create item class

### PR DESCRIPTION
**Pull request by petushka1**
# Setup
### create Item class

- [x] install rubocop
- [x] add linters configuration files
- [x] create and test Item class

### CODE REVIEW

**Review by c3c1l1a — COMMENTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @c3c1l1a, 

Good job so far!
I have a few questions 

## Questions ♻️
_Check the comments under the review._

## Highlights
✅ You set Gitflow correctly
✅ No linter error

File: item.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+require 'date'
+
+# Main super class
+class Item
+  attr_accessor :id, :publish_date, :archived
+
+  def initialize(publish_date)
+    @id = Random.rand(1..1000)
+    @publish_date = Date.parse(publish_date)
+    @archived = false
```
Comment (c3c1l1a) [inline item.rb:10]: - [ ]  There are a few more properties to be declared in the Item class, based on this requirement --> 
`All Item class properties visible in the diagram should be defined and set up in the constructor method. Exception: properties for the 1-to-many relationships should NOT be set in the constructor method. Instead, they should have a custom setter method created.`

image

Comment (petushka1) [reply item.rb:10]: Hey @c3c1l1a thanks for your review. I'd appreciate your input. Added these attributes inside the block outside initializing attributes to allow adding these options from child classes `@attr << self`

**Review by c3c1l1a — APPROVED** [review]
# Status: Approved! ♻️♻️♻️ 

Hi @petushka1, 

Awesome work. 

## Highlights
✅ You set Gitflow correctly
✅ No linter error

**Pull request merged**